lib/js/composables/hover-state.ts

```diff
@@ -0,0 +1,38 @@
+import { Ref, ref } from 'vue';
+import { isTouchDevice } from '../utils/device';
+
+interface HoverState {
+	hovered: Ref<boolean>;
```

Comment:
`isHovered`
https://bethink.atlassian.net/wiki/spaces/IT/pages/1367539761/Konwencje+techniczne#:~:text=Nazwy%20p%C3%B3l%20typu%20boolean%20powinny%20zaczyna%C4%87%20si%C4%99%20od%20is/are/has/have/can/should.